server_addon/max/client/ayon_max/plugins/load/load_pointcache.py

```diff
@@ -31,6 +31,8 @@ def load(self, context, name=None, namespace=None, data=None):
         file_path = self.filepath_from_context(context)
         file_path = os.path.normpath(file_path)
 
+        set_fps()
```

Comment:
Wait a second - this doesn't seem like the right fix? You don't want to force change the FPS of your scene on loading something?

Reply:
But the fps value does change the custom frame range of the alembic object. let's say you have exported the alembic with 1001 to 1010 as animation range. And when you load back the point cache into the scene, it will be loaded into frame range between 1201 to 1210, which are 200 frames more than the original export frame range. 
And wIth the fix here, the point cache loaded in the current animation range. 
Would be great if you can suggest any better solution to fix it.

Reply:
Interesting - so where does the shift in range come from? Is it solely the fact that your scene's FPS is different? (Since that's all you're setting?)

A difference from 1001 to 1201 start frame - is that a 25 FPS / 30 FPS difference issue?

Could you make a screen recording of the issue itself? Unfortunately I know to little of the issue and 3DsMax in general to provide good pointers in this area.

I'd say that if the issue is only on LOAD then we should instead use the FPS of the loaded product (not that of the current task) and then maybe set it temporarily only during the load:

In the simplest form something along the lines of:
```python
product_fps = float(context["version"]["attrib"].get("fps")
if product_fps is None:
    # Just stick to current scene FPS
    product_fps = current_fps()

with scene_fps(product_fps):
    load_file(alembic)
```